controllers/front/AttachmentController.php

```diff
@@ -44,7 +45,45 @@ public function postProcess()
         header('Content-Length: '.filesize(_PS_DOWNLOAD_DIR_.$a->file));
         header('Content-Disposition: attachment; filename="'.utf8_decode($a->file_name).'"');
         @set_time_limit(0);
-        readfile(_PS_DOWNLOAD_DIR_.$a->file);
+        //readfile(_PS_DOWNLOAD_DIR_.$a->file);
+        self::readfile_chunked(_PS_DOWNLOAD_DIR_.$a->file);        
         exit;
     }
+
+    /**
+     * http://ca2.php.net/manual/en/function.readfile.php#54295
+     * jjordan@hofmann.es
+     */
+    function readfile_chunked($filename,$retbytes=true)
+    {
+        // how many bytes per chunk
+        $chunksize = 1*(1024*1024);
+        $buffer = '';
+        $cnt =0;
```

Comment:
I'd like better name for this variable instead of comment explaining how it works, what do you think ?